antsibull/cli/antsibull_build.py

```diff
@@ -126,7 +123,7 @@ def parse_args(program_name: str, args: List[str]) -> argparse.Namespace:
 
     parser = argparse.ArgumentParser(prog=program_name,
                                      description='Script to manage building ACD')
-    subparsers = parser.add_subparsers(title='Subcommands', dest='command',
+    subparsers = parser.add_subparsers(title='Subcommands', dest='command', required=True,
```

Comment:
required was added as a parameter in python-3.7 so we have to set this afterwards instead:

``` python
subparsers = parser.add_subparsers(title='Subcommands', dest='command',
                                   help='for help use antsibull-build SUBCOMMANDS -h')
subparsers.required = True
```

Reply:
Thanks!